vmdb/app/models/bottleneck_event.rb

```diff
@@ -6,7 +6,7 @@ class BottleneckEvent < ActiveRecord::Base
   serialize :context_data
 
   def self.last_created_on(obj)
-    event = self.find(:first, :conditions => ["resource_type = ? AND resource_id = ?", obj.class.name, obj.id], :order => "created_on DESC")
+    event = self.where(["resource_type = ? AND resource_id = ?", obj.class.name, obj.id]).order("created_on DESC").first
```

Comment:
Not now, too many moving pieces.  I started doing that and it's way too easy to get buried in shiny object changes.